impeller/entity/contents/filters/gaussian_blur_filter_contents.cc

```diff
@@ -268,7 +229,7 @@ DownsamplePassArgs CalculateDownsamplePassArgs(
     //
     // The region we cut out will be aligned with the down-sample divisor to
     // avoid pixel alignment problems that create shimmering.
-    int32_t divisor = std::round(1.0f / desired_scalar);
+    int32_t divisor = 1;
```

Comment:
I think this is right, our UVs have to treat the snapshot texture as if its at the regular size, the GPU will give us the higher mip levels.